dask/dataframe/io/parquet/fastparquet.py

```diff
@@ -828,9 +828,8 @@ def read_metadata(
                 "`use_nullable_dtypes` is not supported by the fastparquet engine"
             )
         if kwargs.get("convert_strings", False):
-            warnings.warn(
-                "`dataframe.convert_string` is not supported by the fastparquet engine",
-                UserWarning,
+            raise ValueError(
```

Comment:
@jrbourbeau I intentionally used a warning here and not a `ValueError`, because this kwarg is not explicitly set by the user, but read by the base class from the `dask` config file. The user may set this config setting to `True` for `pyarrow` backend, and then forget to change it and try `fastparquet`, where the setting doesn't play any role. It would be annoying if things break. Do you think a `ValueError` is justified?

Reply:
@jrbourbeau I'd rather revert this back to a warning. What do you think?

Reply:
I don't feel strongly either way, but I suspect that James will favor an error. Either way, be probably want the error message to clarify that the `"dataframe.convert_string"` **config** setting is causing the problem (so the user knows exactly what they need to change).